inspire_classifier/app.py

```diff
@@ -76,6 +82,45 @@ def page_not_found(e):
         }, 404
 
 
+@click.group(cls=FlaskGroup, create_app=create_app)
+def inspire_classifier():
+    "INSPIRE Classifier commands"
+
+
+@inspire_classifier.command('predict-coreness')
+@click.argument('title', type=str, nargs=1)
+@click.argument('abstract', type=str, nargs=1)
+@click.argument('classifier_base_path', type=click.Path(exists=True), required=False, nargs=1)
+def predict(title, abstract, classifier_base_path):
+    """
+    Usage: inspire-classifier predict-coreness TITLE ABSTRACT [CLASSIFIER_BASE_PATH]
```

Comment:
Isn't this message generated automatically by `click`?

Reply:
It is. I removed the docstrings.